HEPML.bib

```diff
@@ -1883,6 +1883,16 @@ @article{Strong:2020mge
       SLACcitation   = "%%CITATION = ARXIV:2002.01427;%%"
 }
 
+% December 27, 2019
+@article{10.1088/2632-2153/ab9023,
+author={George Alverson and CMS Collaboration},
```

Comment:
@vlimant Is this the author field that you actually want? Seems strange to not just have "CMS Collaboration".